packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue

```diff
@@ -68,6 +91,9 @@ const emit = defineEmits(['update:selection']);
 }
 
 .dropdown-option-group {
+	display: flex;
+	gap: 0.2rem;
```

Comment:
```suggestion
gap: var(--gap-xsmall);
```

Reply:
Or I think it'd be nice to have some space in between
```suggestion
   justify-content: space-between;
```

Reply:
change made